corehq/apps/settings/templates/settings/user_api_keys.html

```diff
@@ -78,6 +80,7 @@ <h3>
     <td data-bind="text: ip_allowlist"></td>
     <td data-bind="text: created"></td>
     <td data-bind="text: last_used"></td>
+    <td data-bind="text: expiration_date"></td>
```

Comment:
I'm not sure how easy this is to do but I think the usability of this would be better if it were represented as a 'time' element that displayed the relative time (actual time displayed on hover). Here's an example:

```html
<time title="{{ expiration_date|date:'r' }}"
      datetime="{{ expiration_date.isoformat }}">
  {{ expiration_date|timeuntil }}
</time>
```